parlai/agents/drqa/utils.py

```diff
@@ -49,6 +63,10 @@ def build_feature_dict(opt):
         feature_dict['in_question_uncased'] = len(feature_dict)
     if opt['use_tf']:
         feature_dict['tf'] = len(feature_dict)
+    if opt['use_ner']:
```

Comment:
This is a little weird. You are using one feature each for `pos` and `ner`, and encoding that later as a integer value {0, num_types} ? This should either be a one-hot encoding or used as a low-dimensional embedding.

Reply:
Sorry I misread this last time, but same idea remains. see line 23 comment